api/src/main/resources/liquibase.xml

```diff
@@ -131,5 +131,15 @@
     		<column name="state" />
     	</createIndex>
     </changeSet>
+
+    <changeSet author="pawelgutkowski" id="openconceptlab-v1-11">
+    	<preConditions onFail="MARK_RAN">
+    		<not><columnExists indexName="openconceptlab_item_updated_on"/></not>

```

Comment:
Probably columnName and not indexName. Please try installing the module in openmrs to see if the liquibase changeset is applied without errors (module won't start unless it can be applied)